createDataset.py

```diff
@@ -4,18 +4,144 @@
 import re
 from datetime import datetime
 
-personName = raw_input('Enter your full name: ')
```

Comment:
The reason is that we're asking for the name individually in each of the functions, correct?

Reply:
precisely!